Tools/CISupport/ews-build/steps.py

```diff
@@ -7358,10 +7445,8 @@ def run(self):
             yield self._addToLog('stdio', 'No API key for {} found'.format(RESULTS_DB_URL))
 
         self.command = ['python3', 'Tools/Scripts/compare-static-analysis-results', os.path.join(self.getProperty('builddir'), 'build/new')]
-        self.command += ['--build-output', SCAN_BUILD_OUTPUT_DIR]
-        self.command += ['--archived-dir', os.path.join(self.getProperty('builddir'), 'build/baseline')]
-        self.command += ['--scan-build-path', '../llvm-project/clang/tools/scan-build/bin/scan-build']  # Only generate results page on the second comparison
-        self.command += ['--delete-results']
+        self.command += ['--build-output', SCAN_BUILD_OUTPUT_DIR, '--archived-dir', os.path.join(self.getProperty('builddir'), 'build/baseline')]
+        self.command += ['--scan-build-path', '../llvm-project/clang/tools/scan-build/bin/scan-build', '--delete-results']  # Only generate results page on the second comparison
         if CURRENT_HOSTNAME in EWS_BUILD_HOSTNAMES + TESTING_ENVIRONMENT_HOSTNAMES and self.getProperty('github.base.ref', DEFAULT_BRANCH) == DEFAULT_BRANCH:
```

Comment:
What are we changing here and why?

Reply:
I just moved `'--delete-results'` one line up to be added with the other arguments to be succinct.